addons/bounty/Bounty_redeem.sqf

```diff
@@ -36,14 +36,21 @@ if(_noMatch)then{
     name _killer,
     _bountyAmount,
     [
-      "is finally dead!",
-      "has ceased to be!",
-      "has expired and gone to meet their maker",
-      "has kicked the bucket!",
-      "got fucked up!",
-      "is taking a swim with cement boots",
-      "is now a worm buffet",
-      "is taking a dirt nap"
+    "has kicked the bucket!",
+	"got fucked up!",
+	"is taking a swim with cement boots",
+	"is now a worm buffet",
+	"is taking a dirt nap",
+	"became a root inspector",
+	"is checking out the grass from underneath",
+	"is deader than a doornail",
+	"is headed home in a pine box",
+	"is out of their misery",
+	"is sleeping with the fishes",
+	"was WASTED",
+	"is pushing daisies",
+	"got whacked",
+	"is deader than Prince"
```

Comment:
Never too soon. Also I'm not sure what's up with the space of line 39 but it looked fine in N++.